pyproject.toml

```diff
@@ -25,9 +25,9 @@ keywords = ['exasol', 'udf', 'script-languages']
 
 [tool.poetry.dependencies]
 python = ">=3.10,<4"
-exasol-script-languages-container-tool = "^1.1.0"
-exasol-script-languages-container-ci = "^1.6.0"
-exasol-script-languages-container-ci-setup = "^1.6.0"
+exasol-script-languages-container-tool = "^2.0.0"
+exasol-script-languages-container-ci = "^1.7.0"
+exasol-script-languages-container-ci-setup = "^1.7.0"
```

Comment:
There is a major version mismatch between the slc-tool, slc-ci and slc-ci-setup. Is this OK? Should the version be aligned?